src/referentielijsten_client/steps.py

```diff
@@ -0,0 +1,61 @@
+from django.core.exceptions import ValidationError
+
+import structlog
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+from zgw_consumers.models import Service
+
+from openklant.config.models import ReferentielijstenConfig
+from referentielijsten_client.models import ReferentielijstenConfigurationModel
+
+logger = structlog.stdlib.get_logger(__name__)
+
+
+class ReferentielijstenConfigurationStep(
```

Comment:
Can move this to `src/referentielijsten_client/setup_configuration/steps.py` and also add `src/referentielijsten_client/setup_configuration/tests/test_client_config_step.py` to make sure this step works as expected? You can see an example of how tests are made for this in `src/openklant/setup_configuration/tests/test_token_auth_config.py`

I think the following two tests are the only necessary ones:
- happy flow where the `selectielijst_api_service_identifier` points to an existing Service in the database -> assert that the config is created with the configured values
- errors should be raised if the `selectielijst_api_service_identifier` does not point to an existing Service
- running the same step twice should be idempotent